src/asset-swapper/utils/market_operation_utils/constants.ts

```diff
@@ -2834,8 +2840,13 @@ export const DEFAULT_GAS_SCHEDULE: Required<GasSchedule> = {
     [ERC20BridgeSource.SpiritSwap]: uniswapV2CloneGasSchedule,
     [ERC20BridgeSource.SpookySwap]: uniswapV2CloneGasSchedule,
     [ERC20BridgeSource.Yoshi]: uniswapV2CloneGasSchedule,
-    [ERC20BridgeSource.Beethovenx]: () => 100e3,
-
+    [ERC20BridgeSource.Beethovenx]: (fillData?: FillData) => {
+        const balancerFillData = fillData as BalancerV2BatchSwapFillData;
+        if (balancerFillData.chainId === ChainId.Fantom) {
+            return 125e3 + ((fillData as BalancerV2BatchSwapFillData).swapSteps.length - 1) * 50e3;
```

Comment:
nit: `balancerFillData.swapSteps`

Reply:
I think we need the type assertion here since the DEFAULT_GAS_SCHEDULE as it stands right now accepts a very generic `FillData` per bridge source rather than source specific FillData type. I think we'd need to define some sort of source to fill data type mapping if we want to avoid the use of type assertions here.